docs/sphinx/reference-frontend-api.rst

```diff
@@ -147,10 +147,10 @@ Structures/Enumerations
    - **OBS_FRONTEND_EVENT_VIRTUALCAM_STOPPED**
 
      Triggered when the virtual camera is stopped.
-   
+
    - **OBS_FRONTEND_EVENT_TBAR_VALUE_CHANGED**
 
-     Triggered when the transition bar is moved. 
+     Triggered when the transition bar is moved.
```

Comment:
Unrelated whitespace changes.

Reply:
Will fix. This was automatically added by VSCode as I've set it up to format the documents on save.